dev-util/clippy/clippy-9.1.ebuild

```diff
@@ -0,0 +1,58 @@
+# Copyright 2020-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+MY_P="frr-${PV}"
+PYTHON_COMPAT=( python3_{9..12} )
+inherit autotools python-single-r1
+
+DESCRIPTION="Standalone clippy tool built from FRR sources"
+HOMEPAGE="https://frrouting.org/"
+SRC_URI="https://github.com/FRRouting/frr/archive/${MY_P}.tar.gz"
+S="${WORKDIR}/frr-${MY_P}"
+
+LICENSE="GPL-2"
```

Comment:
Is this correct? At least on top of clippy, I see:

```
// SPDX-License-Identifier: GPL-2.0-or-later
```

Does it link some non-"or-later" files?

Reply:
I absoluty don’t know. This is what the doc says:

> License for Contributions[](https://docs.frrouting.org/projects/dev-guide/en/latest/workflow.html#license-for-contributions)
> 
> FRR is under a “GPLv2 or later” license. Any code submitted must be released under the same license (preferred) or any license which allows redistribution under this GPLv2 license (eg MIT License). It is forbidden to push any code that prevents from using GPLv3 license. This becomes a community rule, as FRR produces binaries that links with Apache 2.0 libraries. Apache 2.0 and GPLv2 license are incompatible, if put together. Please see http://www.apache.org/licenses/GPL-compatibility.html for more information. This rule guarantees the user to distribute FRR binary code without any licensing issues.

Reply:
I applied @thesamesam suggestion with `LICENSE="GPL-2+"`